pkg/network/encoding/marshal/format.go

```diff
@@ -39,6 +39,19 @@ func (ipc ipCache) Get(addr util.Address) string {
 	return v
 }
 
+func mergeDynamicTags(dynamicTags ...map[string]struct{}) (out map[string]struct{}) {
+	for _, tags := range dynamicTags {
+		if out == nil {
+			out = tags
+			continue
+		}
+		for k, v := range tags {
+			out[k] = v
+		}
+	}
+	return
```

Comment:
nit: redundant return

Reply:
The naked `return` returns the named return value `out` so it can't be omitted AFAICS.

Reply:
true, missed that